.github/workflows/deployment-test.yml

```diff
@@ -67,25 +67,24 @@ jobs:
           path: testnet-load-generator
           ref: ${{steps.get-loadgen-branch.outputs.result}}
 
-      - name: Move repos under /usr/src where scripts expect them
+      - name: Put repos under /usr/src where scripts expect them
         run: |
           set -e
           sudo mv "$GITHUB_WORKSPACE/testnet-load-generator" /usr/src/testnet-load-generator
-          sudo mv "$GITHUB_WORKSPACE" /usr/src/agoric-sdk
+          sudo cp -a "$GITHUB_WORKSPACE" /usr/src/agoric-sdk
```

Comment:
Interesting, why is this needed? And will the cache still be populated?

Reply:
The problem this is solving is that putting the output of the chaintest in the build cache (`"$GITHUB_WORKSPACE"`) would interfere with a rerun of the deployment job (files already existed in the results directory).

So, copying the build outputs into `/usr/src/agoric-sdk` means that the chaintest always has a clean state to start from and can modify anything without it being persisted in the build cache.

Reply:
Did not think of that! We need tests for our test infra 🙃